File: length-of-last-word/sol.go
package main

func lengthOfLastWord(s string) int {
	count := 0
	lenght := len(s)
	start := lenght - 1
	for start >= 0 {
		if s[start] == ' ' {
			start--
			continue
		}
		break
	}
	for start >= 0 {
		if s[start] != ' ' {
			count++
			continue
		}
		break
	}
	return count
}

File: length-of-last-word/sol.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        count = 0
        length = len(s)
        start = length - 1
        while start >= 0:
            if s[start] == ' ':
                start -= 1
                continue
            break

        while start >= 0:
            if s[start] != ' ':
                count += 1
                start -= 1
                continue
            break
        return count
